### PR TITLE
Rock Paper Jonathan Fallas

### PR DESCRIPTION
**Pull request by Elfjhon**
This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

### CODE REVIEW

File: web_development_101/javascript_basics/project_rock_paper_scissors.md [hunk 1]
```diff
@@ -1510,4 +1510,5 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
 - [ebergson's Solution](https://github.com/ebergson/rock-paper-scissors)
 - [Tegristh's Solution](https://github.com/Tegristh/RPS)
 - [jingjing142's Solution](https://github.com/jingjing142/rock-paper-scissors) - [View in Browser](https://jingjing142.github.io/rock-paper-scissors/)
+- [ Jonathan Fallas's Solution](https://github.com/Elfjhon/RockPaper) -[View in Browser](https://elfjhon.github.io/RockPaper/)
```
Comment (I3uckwheat) [inline web_development_101/javascript_basics/project_rock_paper_scissors.md:1513]: Notice the space after the `[` Please be precise when developing. This is what TOP will teach you. Which is a free course that WILL make you a developer. Anybody paying for it, should really re-consider.

**Comment by Elfjhon** [comment]
Done guys thanks

File: web_development_101/javascript_basics/project_rock_paper_scissors.md (hunk 1 above)
Comment (Elfjhon) [reply web_development_101/javascript_basics/project_rock_paper_scissors.md]: thanks for your advice, done

File: web_development_101/javascript_basics/project_rock_paper_scissors.md [hunk 2]
```diff
@@ -1515,6 +1515,7 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
 - [ebergson's Solution](https://github.com/ebergson/rock-paper-scissors)
 - [Tegristh's Solution](https://github.com/Tegristh/RPS)
 - [jingjing142's Solution](https://github.com/jingjing142/rock-paper-scissors) - [View in Browser](https://jingjing142.github.io/rock-paper-scissors/)
+- [Jonathan Fallas's Solution](https://github.com/Elfjhon/RockPaper) - [View in Browser](https://elfjhon.github.io/RockPaper)
```
Comment (marvingay) [inline web_development_101/javascript_basics/project_rock_paper_scissors.md:1518]: I corrected your markdown, however, the live link to your solution (corresponding to `View in browser`) goes to a blank page with just a header.

Verify that the page shows what is intended for your solution and comment back.

Comment (Elfjhon) [reply web_development_101/javascript_basics/project_rock_paper_scissors.md:1518]: Hi Marvin I already made the changes, the code is running pretty good now.
Thanks for your feedback.

**Pull request merged**